NOTES.rst

```diff
@@ -33,6 +30,20 @@ Deprecation warnings
              analogous to configuration of SNMP-based fetching.  :ref:`See below
              for more details<5.12-new-http-rest-api-management-profile-type>`.
 
+Change ``ip2mac`` plugin order in :file:`ipdevpoll.conf`
+--------------------------------------------------------
+
+The Palo Alto ARP plugin in ipdevpoll had a problem which caused the ARP
+records it collected from Palo Palo firewalls to be unduly closed by the
+regular SNMP-based ARP plugin.  This release of NAV fixes this by making the
+SNMP-based ARP plugin a "fallback" mechanism that doesn't touch ARP collection
+if another plugin has already collected ARP data.
+
+In order for this fix to work, **you must change the order of the plugins** in the
```

Comment:
For a follow-up PR you could make a validation error if this is spotted when reading the config file.

Reply:
.. you could even reshuffle the order in memory and still complain about the order.

Reply:
Both of those could be an option, but users are free to add any plugin they like in any order they like in the config file, so it's hard for us to make hard and fast rules in code about re-ordering specific plugins without mucking up a potential order that was specifically desired by the users.

At best, IMHO, we could make a *warning* if the `arp` plugin is configured to run before the `paloaltoarp` plugin, but that's about it.  The problem is that warnings will often be suppressed when starting a NAV daemon, and would only be visible in the logs, for those who care to look at them.

Reply:
Added #3256 to remember this